sandbox/apparmor/apparmor.go

```diff
@@ -95,6 +96,10 @@ func setupConfCacheDirs(newrootdir string) {
 		}
 	}
 	SnapConfineAppArmorDir = filepath.Join(dirs.SnapdStateDir(newrootdir), "apparmor", snapConfineDir)
+
+	if err := osutil.EnsureFileState(InternalAbi30File, &osutil.MemoryFileState{Content: []byte(Abi30Text), Mode: 0o644}); err != nil {
```

Comment:
don't we have a place to do this that is not inside an init?

Reply:
We could make this explicit but I noticed that we already have an init that does a few things. This has to work both from snapd-apparmor and from inside snapd proper. It does not make sense to move to interface backend setup code entirely.

We could make the function explicit and then call it from apparmor backend setup code and from snapd-apparmor main but I don't know if that is really better. Do you have a preference?

Reply:
I think I do prefer a function that is potentially called by a couple of places. Unless you find another place where we write to disk from an init? I hope we don't have one